### PR TITLE
Updates trove to install with corrent user/group

### PR DESCRIPTION
**Pull request by dguitarbite**
Backports PR https://github.com/crowbar/barclamp-trove/pull/26

### CODE REVIEW

**Comment by dguitarbite** [comment]
Wrong branch! Sorry for the spam

**Comment by dirkmueller** [comment]
That fixes the problem only for new users, not for existing ones. Did you also submit a chown patch somewhere?

**Comment by dguitarbite** [comment]
No, I will submit a patch for existing ones. Thanks for the update.

**Comment by tboerger** [comment]
LGTM

File: chef/cookbooks/openstack-database/recipes/api.rb [hunk 1]
```diff
@@ -32,6 +32,12 @@ class ::Chef::Recipe # rubocop:disable Documentation
   package pkg
 end
 
+file '/var/log/trove/trove-manage.log' do
+    owner node['openstack']['database']['user']
+    group node['openstack']['database']['group']
+    mode 00700
```
Comment (dirkmueller) [inline chef/cookbooks/openstack-database/recipes/api.rb]: only_if { node["platform"] == "suse" }

since this is a suse specific bug

**Comment by dirkmueller** [comment]
LGTM

**Comment by vuntz** [comment]
Can you forward-port this patch to master?

**Comment by dguitarbite** [comment]
Yes, I will forward-port it.

**Pull request merged**